### DISCUSSION TITLE
如何把正文的标题的英文字体改为Times New Roman

### DISCUSSION THREAD

**Discussion by phyzhenli** [stone-zeng/fduthesis#313] [Q&A] [answered]
请问有无办法把正文的标题的英文字体改为Times New Roman？

原先：
![2023-11-16_18-30](https://github.com/stone-zeng/fduthesis/assets/48823046/e0f890b3-c1cc-45fc-87c7-0413d694dedf)

期望：
![2023-11-16_18-30_1](https://github.com/stone-zeng/fduthesis/assets/48823046/d6b2c9b6-53ef-45ee-b510-cba13fbc0df0)

**Comment by sikouhjw** [comment, marked as answer]
https://github.com/stone-zeng/fduthesis/blob/d84e0dd7df657d47943e98c98ff843cc1336bc5f/source/fduthesis.dtx#L3439-L3470
把
```tex
     \@@_setsansfont:nn { texgyreheros } 
       { 
         Extension      = .otf, 
         UprightFont    = *-regular, 
         BoldFont       = *-bold, 
         ItalicFont     = *-italic, 
         BoldItalicFont = *-bolditalic 
       } 
```
换成
```tex
     \@@_setsansfont:nn  { XITS } 
       { 
         Extension      = .otf, 
         UprightFont    = *-Regular, 
         BoldFont       = *-Bold, 
         ItalicFont     = *-Italic, 
         BoldItalicFont = *-BoldItalic, 
       } 
```
注意，这是很丑陋的字体搭配，指中文无衬线+西文衬线。
